Cargo.toml

```diff
@@ -151,9 +151,7 @@ reqwest = { version = "0.12.5", default-features = false, features = [
     "stream",
 ] }
 rlimit = { version = "0.10.1" }
-rocksdb = { version = "0.22.0", features = [
-    "multi-threaded-cf",
-], git = "https://github.com/restatedev/rust-rocksdb", rev = "c7ccbbcd261bdec011c4976c441676512a1a4841" }
+rust-rocksdb = { version = "0.29.0", features = ["multi-threaded-cf"], git = "https://github.com/restatedev/rust-rocksdb", rev = "8b621f1b4fb15ed0ba4a4b9da239432159e40b01" }
```

Comment:
It'd be less disruptive if you specify `package = "rust_rocksdb"` to avoid the mass-rename from `rocksdb to `rust_rocksdb`. 

example:
```
rocksdb = { version = "0.29.0", features = ["multi-threaded-cf"], git = "https://github.com/restatedev/rust-rocksdb", rev = "8b621f1b4fb15ed0ba4a4b9da239432159e40b01", package = "rust_rocksdb" }
```

Reply:
That's true. I wanted to rename it because I find it very confusing if there is somewhere in the chain a rename from "a" to "b". If one misses this one rename, then it can be quite confusing where things are coming from.

Reply:
I understand that this can be confusing. However, it'll still be called "rocksdb" in call of our cargo files. My motivation on keeping it as `rocksdb` stems from the fact that `rust-*` as a name is an anti-pattern, and that we might need to go back and forth between those forks, or even maintain our own and we wouldn't need to rename everything, everytime.

Reply:
Definitely in favor of keeping the `rocksdb` as the package name / prefix if it's this easy to achieve.

Reply:
I've updated the PR to remove the renaming.